### PR TITLE
bump(*)

### PR DESCRIPTION
**Pull request by coreydaley**
Includes bump due to https://github.com/openshift/api/pull/285
changes to use go-bindata/go-bindata
updates to use klog

### CODE REVIEW

**Comment by coreydaley** [comment]
/assign @adambkaplan 

New pull request to bump deps

File: glide.lock [hunk 1]
```diff
@@ -202,7 +202,7 @@ imports:
 - name: github.com/NYTimes/gziphandler
   version: 56545f4a5d46df9a6648819d1664c3a03a13ffdb
 - name: github.com/openshift/api
-  version: fe0c0ad070ce15d30b0a6dd20adb0ef883efe11a
+  version: e8532c51d46fdc99060e3357da9282a3c588ee94
```
Comment (adambkaplan) [inline glide.lock]: sha is almost a month old - did you clear your glide cache?

**Comment by coreydaley** [comment]
/retest 

aws_vpc.new_vpc: Error creating VPC: VpcLimitExceeded: The maximum number of VPCs has been reached.

File: glide.lock (hunk 1 above)
Comment (coreydaley) [reply glide.lock]: must not have, updating.

**Comment by coreydaley** [comment]
@adambkaplan ptal

**Comment by coreydaley** [comment]
@adambkaplan ptal

**Comment by coreydaley** [comment]
/retest

failed to initialize the cluster: Working towards 0.0.1-2019-04-11-160713: 91% complete: timed out waiting for the condition

File: glide.yaml [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/kube-aggregator
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/code-generator
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/apiextensions-apiserver
-  repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
```
Comment (deads2k) [inline glide.yaml]: check this and the next against kas-o

File: glide.yaml [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
 - package: github.com/openshift/api
   version: master
 - package: github.com/openshift/client-go
   version: master
 - package: github.com/openshift/library-go
   version: master
+  #others last
 - package: github.com/jteeuwen/go-bindata
   version: a0ff2567cfb70903282db057e799fd826784d41d
 - package: k8s.io/gengo/args
```
Comment (deads2k) [inline glide.yaml]: check against kaso

Comment (coreydaley) [reply glide.yaml]: updated, thanks

File: glide.yaml (hunk 2 above)
Comment (coreydaley) [reply glide.yaml]: updated, thanks

**Comment by adambkaplan** [comment]
/hold

Other control plane operators are not on golang 1.11

**Comment by coreydaley** [comment]
@adambkaplan ptal, dropped the go1.11 commit

**Comment by coreydaley** [comment]
/retest



could not copy stable imagestreamtag: Timeout: request did not complete within allowed duration

**Comment by coreydaley** [comment]
/hold
needs another dependency bump when library-go pr has merged https://github.com/openshift/library-go/pull/354

**Comment by coreydaley** [comment]
@adambkaplan ptal, I think I got it this time, the last failure was a cluster installation issue

**Pull request merged**